proto/user.proto

```diff
@@ -65,6 +65,7 @@ message GrantPrivilege {
     uint32 all_tables_schema_id = 11;
     uint32 all_sources_schema_id = 12;
     uint32 all_dml_tables_schema_id = 13;
+    uint32 subscription_id = 14;
```

Comment:
FYI, non-related to this PR, we also have to define all available acls for subscription in https://github.com/risingwavelabs/risingwave/blob/be30f20e2c14b5d12efd5b33a4ac9e537602fc37/src/common/src/acl/mod.rs and support privilege related syntax for subscription. We can support it in an separate PR after the functionality of subscription is ready.